src/commons/layouts/base/Container/Container.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import cn from "classnames";
+import styles from './Container.module.css'
+
+interface Props extends React.PropsWithChildren{
+}
+
+/**
+ * Container layout
+ *
+ * @param props - props of component
+ */
+const Container: React.FC<Props> = ({ ...props }) => {
```

Comment:
You can take only children here:

```suggestion
const Container: React.FC<Props> = ({ children }) => {
```